exchange/exchange-ps/exchange/Set-RetentionCompliancePolicy.md

```diff
@@ -663,9 +676,13 @@ The DeletedResources parameter specifies the deleted Microsoft 365 Group, mailbo
 
 A valid value is a JSON string. Refer to the Examples section for syntax and usage examples of this parameter.
 
-For more information on the deleted Microsoft 365 Group scenario, see [Learn more about modern group deletion under retention hold](https://learn.microsoft.com/purview/retention-settings#what-happens-if-a-microsoft-365-group-is-deleted-after-a-policy-is-applied).
+**CAUTION**: When you use a SMTP address with this parameter, be aware that the same address might also be in use for other mailboxes or mail users. To check for additional mailboxes or mail users with the same SMTP address, use the following command and replace *user@contoso.com* with the SMTP address to check: `Get-Mailbox -RecipientTypeDetails UserMailbox,SharedMailbox | Where-Object { $_.EmailAddresses -contains "SMTP:user@contoso.com" } | Select-Object DisplayName, EmailAddresses, Description, Alias, OrganizationalUnit`
+
+To prevent other mailboxes or mail users with the same SMTP address from being excluded, put the mailbox on [Litigation Hold](https://learn.microsoft.com/purview/ediscovery-create-a-litigation-hold) before you run the command with the DeletedResources parameter.
```

Comment:
"to prevent active mailbox or mailuser..."
An object that is already deleted cannot be re-added from customer side to hold. They can add to hold active users.